<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeFigure</class>
 <widget class="QWidget" name="MakeFigure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Make Figure</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="create_figure_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Figure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_coord_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_coord_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_dot_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Dot</string>
    </property>
   </widget>
   <widget class="QComboBox" name="figures_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="x_coord_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x coord</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_coord_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y coord</string>
    </property>
   </widget>
   <widget class="QLabel" name="figure_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>figures list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
